### DISCUSSION TITLE
Why active is a fixed true?

### DISCUSSION THREAD

**Discussion by ir4y** [hl7au/au-fhir-core#43] [Consensus Discussions for R1]
I have a question about the fixed value for the `active` field in Patient, Practitioner, PractitonerRole, and Organization.
Could you please share a bit more details on why it is a fixed true value? 

There are multiple use cases when you need to make a resource inactive. @heathfrankel mentioned some of them in the GitHub discussion (please check related topics for dedicated resources) and all of them make sense to me.

**Comment by dt-r** [comment]
Hi @ir4y, I will copy this feedback from you onto each of the consensus discussions for those resources so it is tracked as part of the resource feedback and close this question.

The response to the question on why is:
_the value of active in the current draft for engagement has it's origins in source material profiles. These are exclusively exchange profiles. I encourage you to add to the feedback here: https://github.com/hl7au/au-fhir-core/discussions/categories/consensus-discussions-for-r1. We'll be working through that feedback in the [September 28 AU Core Workshop](https://confluence.hl7.org/pages/viewpage.action?pageId=184920587)._
